scst-store/amr/auth-k8s-sa-autoconfiguration.md

```diff
@@ -1,24 +1,23 @@
 # Kubernetes service account autoconfiguration
 
-This site is giving an overview about the resources that get created for the convenience feature of kubernetes service account autoconfiguration for AMR authentication and authorization. It will help you nderstand which resource play a role in the default configuration and where to look at in case issues arise. If you need a more deep understanding of the resources and if you are in need to set up these resources on your own, for example if you use means to manage your service accounts or have other requirements related to roles and bindings head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md)
+This site gives an overview of the resources created for the convenience feature of kubernetes service account autoconfiguration for AMR authentication and authorization. It will help you understand which resource plays a role in the default configuration and where to look in case issues arise. If you need a deeper understanding of the resources and if you are in need to set up these resources on your own, for example, if you use means to manage your service accounts or have other requirements related to roles and bindings head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md)
```

Comment:
```
If you need a deeper understanding of the resources and if you are in need to set up these resources on your own, for example, if you use means to manage your service accounts or have other requirements related to roles and bindings head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md)
```

This sentence is a little awkward. What do you think about simplifying it down to something like: 

```
If you need a deeper understanding of the resources or if you need to set up these resources, head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md). 
```